### PR TITLE
Add a DateTimeOffset variable to PlayerLeaveEventArgs

### PR DESCRIPTION
**Pull request by paricbat**

### CODE REVIEW

**Comment by Craftplacer** [comment]
I unfortunately don't see any reason why this is an useful change, events are supposed to be handled immediately by the code listening to it. When a player left, the code is basically executing as soon when it happened.

Can you give me/us a reason why this might be useful?

**Comment by Seb-stian** [comment]
> Can you give me/us a reason why this might be useful?

We talked about this on discord. The reason is to stay consistent with `PlayerJoinEventArgs` that (for some reason?) also has it.

**Comment by Craftplacer** [comment]
> > Can you give me/us a reason why this might be useful?
> 
> We talked about this on discord. The reason is to stay consistent with `PlayerJoinEventArgs` that (for some reason?) also has it.

Really weird. Well, the code seems fine and simple to me, I'll approve it.

File: Obsidian.API/Events/PlayerLeaveEventArgs.cs [hunk 1]
```diff
@@ -1,7 +1,17 @@
...
-        public PlayerLeaveEventArgs(IPlayer player) : base(player) { }
+        /// <summary>
+        /// The date the player left.
+        /// </summary>
+        public DateTimeOffset LeaveDate { get; }
+		
+        public PlayerLeaveEventArgs(IPlayer player, DateTimeOffset leave) : base(player)
+		{
+			this.LeaveDate = leave;
+		}
```
Comment (Seb-stian) [inline Obsidian.API/Events/PlayerLeaveEventArgs.cs]: Indentation seems off?

Comment (Craftplacer) [inline Obsidian.API/Events/PlayerLeaveEventArgs.cs]: Wrong identation

**Review by Seb-stian — APPROVED** [review]

**Review by Craftplacer — APPROVED** [review]

**Review by Craftplacer — APPROVED** [review]

**Pull request merged**